<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberCtrlMaskWdg</class>
 <widget class="QWidget" name="MemberCtrlMaskWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>144</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MemberCtrlMaskWdg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	background-color: rgb(102, 102, 102,125);
	border: 2px solid #52CC90;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>77</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_camera">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-image: url(:/vhallActive/img/vhallactive/camera_normal.png);
}
QPushButton:hover{
	border-image: url(:/vhallActive/img/vhallactive/camera_hover.png);
}
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/camera_click.png);
}

	
</string>
          </property>
          <property name="text">
           <string>摄像头</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_mic">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/mic_click.png);
}

QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/mic_hover.png);
}

QPushButton{
border-image: url(:/vhallActive/img/vhallactive/mic_normal.png);
}
</string>
          </property>
          <property name="text">
           <string>麦克风</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_KickOutPublish">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/vhallActive/img/vhallactive/off_speak.png);
}
QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/off_speak_hover.png);
}
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/off_speak_click.png);
}
</string>
          </property>
          <property name="text">
           <string>下麦</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_setPen">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/setspeaker_click.png);
}

QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/setspeaker_hover.png);
}

QPushButton{
border-image: url(:/vhallActive/img/vhallactive/setspeaker.png);
}
</string>
           </property>
           <property name="text">
            <string>画笔</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_fullScreen">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/main_view_click.png);
}

QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/main_view_hover.png);
}

QPushButton{
border-image: url(:/vhallActive/img/vhallactive/main_view.png);
}
</string>
           </property>
           <property name="text">
            <string>全屏</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_floatRender">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>显示窗口</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>77</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
